frontend/src/views/SqlEditor/AsidePanel/SheetContainer.vue

```diff
@@ -85,104 +85,104 @@
   >
     <DeleteHint
-      :content="$t('sql-editor.hint-tips.confirm-to-delete-this-saved-query')"
+      :content="$t('sql-editor.hint-tips.confirm-to-delete-this-sheet')"
       @close="handleHintClose"
-      @confirm="handleDeleteSavedQuery"
+      @confirm="handleDeleteSheet"
     />
   </BBModal>
 </template>
 
 <script lang="ts" setup>
 import { computed, reactive, ref, nextTick } from "vue";
 import { useI18n } from "vue-i18n";
+import { useRouter } from "vue-router";
 import {
   useNamespacedActions,
   useNamespacedGetters,
   useNamespacedState,
 } from "vuex-composition-helpers";
 
 import {
-  EditorSelectorActions,
-  EditorSelectorGetters,
-  EditorSelectorState,
-  SavedQuery,
+  TabActions,
+  TabGetters,
+  TabState,
+  SheetActions,
+  SheetState,
+  Sheet,
   SqlEditorActions,
-  SqlEditorState,
 } from "../../../types";
 import { getHighlightHTMLByKeyWords } from "../../../utils";
 import DeleteHint from "./DeleteHint.vue";
 
 interface State {
   search: string;
   isShowDeletingHint: boolean;
-  currentSavedQueryName: string;
-  editingSavedQueryId: number | null;
-  currentActionSavedQuery: SavedQuery | null;
+  currentSheetName: string;
+  editingSheetId: number | null;
+  currentActionSheet: Sheet | null;
 }
 
 const { t } = useI18n();
+const router = useRouter();
 
-const { savedQueryList, isFetchingSavedQueries: isLoading } =
-  useNamespacedState<SqlEditorState>("sqlEditor", [
-    "savedQueryList",
-    "isFetchingSavedQueries",
-  ]);
-const { queryTabList } = useNamespacedState<EditorSelectorState>(
-  "editorSelector",
-  ["queryTabList"]
-);
-const { currentTab } = useNamespacedGetters<EditorSelectorGetters>(
-  "editorSelector",
-  ["currentTab"]
-);
-const { deleteSavedQuery, setShouldSetContent, patchSavedQuery } =
+const { sheetList, isFetchingSheet: isLoading } =
+  useNamespacedState<SheetState>("sheet", ["sheetList", "isFetchingSheet"]);
+const { tabList } = useNamespacedState<TabState>("tab", ["tabList"]);
+const { currentTab } = useNamespacedGetters<TabGetters>("tab", ["currentTab"]);
+
+// actions
+const { setShouldSetContent, setActiveConnectionByTab } =
   useNamespacedActions<SqlEditorActions>("sqlEditor", [
-    "deleteSavedQuery",
     "setShouldSetContent",
-    "patchSavedQuery",
+    "setActiveConnectionByTab",
   ]);
-const { updateActiveTab, setActiveTabId } =
-  useNamespacedActions<EditorSelectorActions>("editorSelector", [
+const { updateActiveTab, setActiveTabId, addTab } =
+  useNamespacedActions<TabActions>("tab", [
     "updateActiveTab",
     "setActiveTabId",
+    "addTab",
   ]);
+const { deleteSheet, patchSheetById } = useNamespacedActions<SheetActions>(
+  "sheet",
+  ["deleteSheet", "patchSheetById"]
+);
 
 const state = reactive<State>({
   search: "",
-  currentSavedQueryName: "",
-  editingSavedQueryId: null,
+  currentSheetName: "",
+  editingSheetId: null,
   isShowDeletingHint: false,
-  currentActionSavedQuery: null,
+  currentActionSheet: null,
```

Comment:
if currentSheetName refers to the currentActionSheet, then they should use the same format.

Not one is currentXXX, the other one is currentAction

If they are different, then we should try to find less confusing names, or as the last resort, add comment.